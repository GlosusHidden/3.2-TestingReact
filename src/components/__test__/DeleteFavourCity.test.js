import React from 'react'
import renderer from 'react-test-renderer'

import { Provider } from 'react-redux'

import DeleteFavourCity from '../DeleteFavourCity'

import {myStore, mockStore} from './test_data/data'

it('DeleteFavourCity Component Snapshot', () => {
  const renderedValue = renderer.create(
    <Provider store={mockStore}>
      <DeleteFavourCity id={1}/>
    </Provider>
  ).toJSON();
  expect(renderedValue).toMatchSnapshot();
});
