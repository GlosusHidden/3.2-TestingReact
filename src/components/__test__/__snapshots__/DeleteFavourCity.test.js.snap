// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteFavourCity Component Snapshot 1`] = `
<input
  alt="delete"
  className="img_button"
  name="submit"
  onClick={[Function]}
  src="img/del.png"
  type="image"
/>
`;
